src/main/java/seedu/address/model/task/Task.java

```diff
@@ -111,7 +111,8 @@ public String toString() {
         return new ToStringBuilder(this)
                 .add("description", description)
                 .add("deadline", deadline)
-                .add("event", associatedEvent.getName())
+                .add("event", associatedEventName)
+                .add("isDone", isCompleted)
```

Comment:
Maybe we can just use "isCompleted" as the title to standandise across the project?

Reply:
Good suggestion!